### PR TITLE
Removing @SkipForRepeat for Jakarta REST 3.0 tests.

### PR DESCRIPTION
**Pull request by andymc12**
- many tests just work now
- some updates to tests required for different implementation
- implemented app security autofeature
- more detailed comments when tests legitimately should be skipped
- cleaned up several warnings - particularly deprecated Apache HTTP Client APIs

### CODE REVIEW

**Comment by andymc12** [comment]
#build

**Comment by andymc12** [comment]
#libby

**Comment by andymc12** [comment]
Updated auto feature symbolic name to match filename (removed an extra dash) - new #build

**Comment by andymc12** [comment]
A few more tweaks (and a few more test cases) - new #build

**Comment by andymc12** [comment]
Hopefully the last tweak... #build

**Comment by andymc12** [comment]
auto build command didn't seem to work... so kicked off build in RTC: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_924dYM8PEeu6B9yc149Q8g

File: dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/params/ParamsTest.java [hunk 1]
```diff
@@ -50,18 +53,19 @@
 import componenttest.annotation.Server;
 import componenttest.annotation.SkipForRepeat;
 import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.JakartaEE9Action;
 import componenttest.topology.impl.LibertyServer;
 
 @RunWith(FATRunner.class)
-@SkipForRepeat("EE9_FEATURES") // currently broken due to multiple issues
+//@SkipForRepeat("EE9_FEATURES") // currently broken due to multiple issues
```
Comment (jim-krueger) [inline dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/params/ParamsTest.java]: Remove?

**Review by jim-krueger — APPROVED** [review]

**Comment by andymc12** [comment]
#libby

**Pull request merged**